src/ckb-daemon/usb_mac.c

```diff
@@ -838,6 +931,8 @@ int usbmain(){
     if(iterator_hid)
         iterate_devices_hid(0, iterator_hid);
 
+    register_mouse_event_tap(mainloop);
```

Comment:
This is a valid approach, as it seems this fix only/mostly targets Sierra. I am for Sierra-guarding. 😄